src/Grok.Net.Tests/UnitTests.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using Xunit;
+using GrokNet;
+
+namespace GrokNetTests
+{
+    public class UnitTests
+    {
+        [Fact]
+        public void ParseEmptyTest()
+        {
+            Grok act = new Grok("");
+            GrokResult result = act.Parse("");
+
+            Assert.NotNull(result);
+            Assert.Empty(result);
+        }
+
+        [Fact]
+        public void PatternCountTest()
```

Comment:
Could you explain the meaning of this test?

Reply:
The Grok pattern contains 8 elements, and the logs contains 2 entries. This test simply validates that the logs are correctly parsed, using the defined pattern, by testing that a total of 16 elements are returned.